package.json

```diff
@@ -29,6 +36,8 @@
     },
     "peerDependencies": {
         "@dhis2/app-runtime": "^2.0.4",
-        "d2": "^31.8.1"
+        "d2": "^31.8.1",
```

Comment:
These could probably be a bit more liberal - `d2@>30` maybe?  And perhaps `^2` for `app-runtime`?  I don't know if there's a minimum version we should enforce for `d2`...

Reply:
Tests failed with d2 ^30. Passed with d2 ^31. So I put it at 31.

Reply:
OK, could be a bit more lenient and leave it at `^31`, but not critical

Reply:
And actually it should be `>31` not `^31` because otherwise it won't match versions `>=32`

Reply:
(though that doesn't exist yet and might never exist... so not a huge deal)

Reply:
Updated to >30